networkx/algorithms/centrality/katz.py

```diff
@@ -327,4 +327,4 @@ def katz_centrality_numpy(G, alpha=0.1, beta=1.0, normalized=True, weight=None):
 
     # Normalize: rely on truediv to cast to float
     norm = np.sign(sum(centrality)) * np.linalg.norm(centrality) if normalized else 1
-    return dict(zip(nodelist, centrality / norm))
+    return dict(zip(nodelist, (centrality / norm).tolist()))
```

Comment:
The comment on line 328 (two lines up) says that `truediv` will cast to a float. Can we change that comment to indicate that truediv casts to float and tolist casts to a python number?  (is "casts" the right word for conversion to  python object? and is "Python number" a thing?  I think it is an official type in python. I want a name for a number object that could be integer or real or complex or etc.  I guessed at "Python number" but if someone knows another name being used please let me know)

perhaps the comment could be something like:
```
    # Normalize: use truediv and tolist to convert to float and python number
```

Reply:
Done, using my own wording.